kai/reactive_codeplanner/task_runner/analyzer_lsp/api.py

```diff
@@ -39,6 +39,7 @@ def fuzzy_equals(self, error2: Task, offset: int = 1) -> bool:
             self.ruleset.name == error2.ruleset.name
             and self.incident.message == error2.incident.message
             and self.file == error2.file
+            and self.incident.variables == error2.incident.variables
```

Comment:
+1

Reply:
Hmm,  none of the ones that I looked through did,  but maybe certain providers do. We'd need to filter it out in that case

Reply:
Can you give me an example really fast of the variables that are compared then?

Reply:
These are the only keys I see, so I think it's safe to merge

```
(['data', 'innerText', 'matchingXML'])
(['file', 'kind', 'name', 'package'])
(['file', 'kind', 'name', 'package', 'renamed'])
(['matchingText'])
```

Reply:
Ok then lets move forward, just know that this could be an issue in the future.